nlp/README.md

```diff
@@ -0,0 +1,22 @@
+## DACCS - NLP docker image
```

Comment:
please change to "PAVICS NLP Docker Image"
DACCS is the project name, but PAVICS is the platform ;)